modules/ardrivo/src/Wire.cxx

```diff
@@ -86,55 +74,77 @@ uint8_t TwoWire::endTransmission(uint8_t)
 {
     if (transmitted > 0)
     {
-        transmitting = false;
+        transmissionAddress.reset();
+        txBufferIndex = 0;
         //0 = sucess transmission
         return 0;
     }
-    transmitting = false;
+    transmissionAddress.reset();
     //Error at transmission
     return 1;
 };
 
+// REQUEST_RECEIVE
 size_t TwoWire::write(uint8_t amount)
```

Comment:
This parameter name is wrong, since this function overload is not writing an amount, but rather a value